subsys/bluetooth/controller/ll_sw/ull_llcp.c

```diff
@@ -71,17 +65,26 @@ static struct mem_pool mem_ctx = { .pool = buffer_mem_ctx };
 /*
  * LLCP Resource Management
  */
-static struct proc_ctx *proc_ctx_acquire(void)
+static struct proc_ctx *proc_ctx_acquire(struct mem_pool *owner)
```

Comment:
Nit-pick: In my humble opinion `owner` could be more general eg `ctx_pool`. That makes the function meaning to be more general. `ctx->owner` points to a source `mem_pool` where the acquired `ctx` should be released.